Sources/Views/MessageInputBar.swift

```diff
@@ -423,6 +449,16 @@ open class MessageInputBar: UIView {
     // MARK: - Notifications/Hooks
     
     @objc open func orientationDidChange() {
+        
```

Comment:
Nit 😛 Can we move the @`objc` up one line?

Reply:
sure